components/NavLinks/index.js

```diff
@@ -0,0 +1,87 @@
+import styled from "styled-components";
+import { StyledLink } from "@/components/StyledLinks";
+import { Icon } from "@/components/Icon";
+import { useRouter } from "next/router";
+
+export default function NavLinks({ getRandomActivity, closeMenu }) {
+  const router = useRouter();
+
+  return (
+    <>
+      <StyledLink href="/">
+        <StyledNavIcon $isActive={router.pathname === "/"}>
+          <Icon name="home" />
+          <StyledSubline onClick={closeMenu}>Home</StyledSubline>
+        </StyledNavIcon>
+      </StyledLink>
+      <StyledLink href="/createActivity">
+        <StyledNavIcon $isActive={router.pathname === "/createActivity"}>
+          <Icon name="add" color="black" fillColor="transparent" />
+          <StyledSubline onClick={closeMenu}>Add</StyledSubline>
+        </StyledNavIcon>
+      </StyledLink>
+      <StyledNavButton
+        type="button"
+        onClick={() => {
+          getRandomActivity();
+          router.push("/spotlight");
+          if (closeMenu) closeMenu();
+        }}
+      >
+        <StyledNavIcon $isActive={router.pathname === "/spotlight"}>
+          <Icon name="random" />
+          <StyledSubline onClick={closeMenu}>Random</StyledSubline>
+        </StyledNavIcon>
+      </StyledNavButton>
+      <StyledLink href="/favorites">
+        <StyledNavIcon $isActive={router.pathname === "/favorites"}>
+          <Icon name="navHeart" />
+          <StyledSubline onClick={closeMenu}>Favorites</StyledSubline>
+        </StyledNavIcon>
+      </StyledLink>
+    </>
+  );
+}
+
+const StyledSubline = styled.div`
+  font-size: 16px;
+  margin-top: 3px;
+  @media (min-width: 768px) {
+    font-size: 24px;
+  }
+`;
+const StyledNavButton = styled.button`
+  all: unset;
+  font-family: var(--styled-link);
+  font-size: 16px;
+`;
+const StyledNavIcon = styled.div`
+  display: flex;
+  align-items: stretch;
```

Comment:
```suggestion
  align-items: center;
```

since you define below AND above 768 px `align-items: center` you should do it here.